iscript/src/iscript/script.py

```diff
@@ -39,6 +39,8 @@ def check_dep_behavior(task, behavior, supported_behaviors):
         behavior = "mac_sign_and_pkg_vpn"
     if behavior == "mac_notarize_single_file" and "mac_single_file" in supported_behaviors:
         behavior = "mac_single_file"
+    if behavior == "mac_notarize_geckodriver" and "mac_geckodriver" in supported_behaviors:
+        behavior = "mac_geckodriver"
```

Comment:
We could change this to a

```
{
    "orig_format": "fallback_format",
    ...
}
```

tyep config and have a single `if` statement, but this works :)

Reply:
Maybe if we add one more case? For now, I'm happy with the if's.